libs/sdk-ui-vis-commons/src/legend/PopUpLegend/LegendDialog.tsx

```diff
@@ -17,13 +18,19 @@ interface ILegendDialogContent {
 const LegendDialogContent: React.FC<ILegendDialogContent> = (props) => {
     const { title, onCloseDialog, children } = props;
 
+    const isZoomed = useIsZoomed(1.2);
```

Comment:
I saw this number in one other file. Can it be exported as some kind of constant from UI Kit? Maybe `ZOOM_TRESHOLD` or maybe just use this number as default value of the hook param? I expect that there will be other places in near future where this handling will be required.